pkg/controller/azdrivernode.go

```diff
@@ -45,6 +46,16 @@ type ReconcileAzDriverNode struct {
 	namespace string
 }
 
+func (r *ReconcileAzDriverNode) getClient() client.Client {
+	return r.client
+}
+func (r *ReconcileAzDriverNode) getAzClient() azClientSet.Interface {
```

Comment:
np: Please insert blank lines between function definitions.

Reply:
np: It would also be good to move these to the end of file (since they're package private) or at least to after global variable definitions.